### ISSUE TITLE
Sporadic vertical shift glitch on Android

### ISSUE THREAD

**Issue by joaqo** [PostHog/posthog-js#2155]
Hi! This works flawlessly except for a sporadic vertical shift glitch on Android. Sporadically the whole screen shifts up and down for a few milliseconds. I think it gets more triggered when I interact with any react navigation component. Could be caused by safe areas, as I think the vertical shift corresponds with the size of the top safe area.

The diagnosis was quite simple, if I install this library the glitch happens on some Android devices (Pixel 6a for example), if I uninstall it the issue is resolved. I think it only happens on some Android devices because on the other devices (older ones) the package doesn't even get installed as the android sdk version requirements are too new.

Attaching my current package.json and a video of the glitch.
[package.json](https://github.com/user-attachments/files/17742556/package.json)

https://github.com/user-attachments/assets/d05040a3-9bc6-43e3-ae64-79dce125fa2f

**Comment by joaqo** [comment]
Yep, just to confirm that I am still seeing this issue. I removed all calls to safe area which greatly improved it, but I am using react navigation, which contains some safe areas I can't access, so I still see the issue though more sporadically.

**Comment by adpadillar** [comment]
just getting this issue too. definitely still present. i'll see if i can provide MRE later.
